matrix/tests/matrix.test.cpp

```
@@ -8,23 +8,23 @@ using namespace consteig;
 // These helper functions allow us to call templated constexpr functions
 template <typename T, Size R, Size C>
 static constexpr Matrix<T, R, C> setRowsTest(const Matrix<T, 1, C> mat[],
-                                             const T val)
+                                             const Size row)
 {
```

Cppcheck (2.20.0)

[style] The function 'csqrt' is never used.

(unusedFunction)